tests/_utils/editor.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * Mock of class that representing a basic, generic editor.
+ *
+ * @see: https://ckeditor.com/docs/ckeditor5/latest/api/module_core_editor_editor-Editor.html
+ */
+export default class Editor {
+	constructor() {
+		this.model = Editor._model;
```

Comment:
why it can't be `this.model = { document: createDocument() }`?

Reply:
https://github.com/ckeditor/ckeditor5-react/pull/52/files/f1ceee7fb77036c982de21b8be171fb1573237e6#diff-178f392effc12f072e31bd8a383f0a50R31

If you mock those properties after initialization the component, call those functions: `modelDoc.on` and `viewDoc.on` won't be detected.